tests/Kernel/config/packages/shopware_app.yaml

```diff
@@ -34,3 +34,4 @@ shopware_app:
       - foo
   customFieldsPath: '../../path/to/customFields.xml'
   cookiesPath: '../../path/to/cookies.xml'
+  paymentsPath: '../../path/to/payments.xml'
```

Comment:
Do we even need the payments path if we have the the class?

Reply:
Yes for payment without any logic like pay on delivery :smile:

Reply:
But then you can still implement the abstract class without adding a pay method

Reply:
You could still implement something like this with the abstract class if you want to check something every night and update the payment status then with your app. But if you don't need any logic and just want to provide a simple payment method like pay on delivery, this would be handled within a new xml file :smile: